### DISCUSSION TITLE
Allow user to edit the default skins

### DISCUSSION THREAD

**Discussion by RaoraRare** [ppy/osu#24158] [Ideas] [answered]
If you edit any of the default skins your changes will be saved but when you change your skin the defaults with go back to the original I think if the user edits any of the default skins they should save even if you change your skin. The skin used argon pro

https://github.com/ppy/osu/assets/120297366/93e6841c-23f6-4b23-9381-5eeba48c6a7e

**Comment by peppy** [comment, marked as answer]
It does save, as a copy.

**Reply by RaoraRare** [reply]
Yep your right, sorry for the inconvenience
